packages/darklang/languageTools/nameResolver.dark

```diff
@@ -0,0 +1,280 @@
+module Darklang =
+  module LanguageTools =
+    module NameResolver =
+
+      type NameResolver = { allowError: Bool }
```

Comment:
Given the current naming, one might expect to take one of these and call `nr.resolveName` on it.
Maybe we should rename to `NameResolutionSettings`?